client/lib/analytics/index.js

```diff
@@ -513,4 +513,9 @@ const analytics = {
 	},
 };
 emitter( analytics );
+
 export default analytics;
+export const ga = analytics.ga;
```

Comment:
What better way to make sure importing works properly than adding a test for it 😉